.gitignore

```diff
@@ -0,0 +1,116 @@
+
+#Mac OS
+*.DS_Store
+
+#PyCharm IDE
+.idea/
+
+# Documentation build files
+doc/_build/
+doc/ast2vec.rst
```

Comment:
Wow this is an artifact from the ancient times... Can you please remove ast2vec from all our projects